pkg/odo/cli/catalog/list/components.go

```diff
@@ -132,7 +127,18 @@ func (o *ListComponentsOptions) Run() (err error) {
 			}
 		}
 
+		if len(o.catalogDevfileList.Items) != 0 {
+			fmt.Fprintln(w, "Odo Devfile Components:")
+			fmt.Fprintln(w, "NAME", "\t", "DESCRIPTION", "\t", "REGISTRY")
+
+			o.printDevfileCatalogList(w, o.catalogDevfileList.Items, "")
```

Comment:
Can't put these lines
```
fmt.Fprintln(w, "Odo Devfile Components:")
fmt.Fprintln(w, "NAME", "\t", "DESCRIPTION", "\t", "REGISTRY")
```

inside `o.printDevfileCatalogList( .. .)` method?

Same for the S2i components case, o.printCatalogList( ... ) method.